mosh_app/mosh_window.js

```diff
@@ -36,7 +36,14 @@ function execMosh() {
 
   // Don't exit fullscreen with ESC.
   terminal.document_.onkeyup = function(e) {
-    if (e.keyCode == 27) {
+    if (e.keyCode == 78 && e.ctrlKey && e.shiftKey){
+      chrome.app.window.create(
```

Comment:
really seems like this should be unified with the existing `newSession` logic in background.js.  but i'm guessing.

Reply:
newSession() is not available in mosh_window.js... we'd need to move that newSession() logic into a tertiary file that both mosh_window and background inherit, right?

Reply:
right, something would need reworking.  @rpwoodbu would be able to provide suggestions in that regard.

Reply:
Good catch, @vapier. You definitely need to use newSession() in background.js, so that it gets the logic to ensure that any pending update is applied in a timely fashion once all windows have been closed. (Also, it is just good hygiene.)

This should do the trick:
```javascript
chrome.runtime.getBackgroundPage(function (bg) {
  bg.newSession();
});
```